aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/ConversionOption.java

```diff
@@ -16,7 +16,8 @@
 package nl.nn.adapterframework.extensions.aspose;
 
 public enum ConversionOption {
-	SINGLEPDF(0), SEPARATEPDF(1);
+    SINGLEPDF(0), SEPERATEPDF(1);
+    // Written as "SEPERATEPDF" instead of "SEPARATEPDF" because Aspose libary expects "SEPERATEPDF"
```

Comment:
Separate is often misspelled as seperate, a word that has no meaning and is simply a misspelling